packages/rpc-core/README.md

```diff
@@ -64,11 +68,12 @@ api.chain
       console.error('error:', error);
     }
 
-    console.log(`best #${header.number.toString()}`);
+    console.log(`best #${header.raw.number.toString()}`);
```

Comment:
This should be `header.blockNumber` rather. (As far as possible, should not encourage access to .raw)